packages/workbox-core/src/controllers/WorkboxCore.js

```diff
@@ -0,0 +1,26 @@
+import assert from '../utils/assert';
+import logHelper from '../utils/logHelper';
+
+/**
+ * WorkboxCore shares code across Workbox modules.
+ */
+export default class WorkboxCore {
+  /**
+   * @private
+   */
+  get INTERNAL() {
+    if (this._internal) {
```

Comment:
I don't feel strongly, but as an alternative to an early `return`, you can consider restructuring this code to match what some of the other getters do, which is to check for the negated value first, and if the value isn't present, initialize it, and then unconditionally return the value at the end.

C.f. https://github.com/GoogleChrome/workbox/blob/master/packages/workbox-cache-expiration/src/lib/cache-expiration.js#L116

Your call.